### ISSUE TITLE
'brace-style' couldn't handle `a.${b.prop}.${c.prop}`; syntax

### ISSUE THREAD

**Issue by omarcinp** [eslint/eslint#7303]
Rule 'brace-style' mark `a.${b.prop}.${c.prop}`; syntax as an error

**Tell us about your environment**
- **ESLint Version:3.6.1**
- **Node Version:6.6.0**
- **npm Version:3.10.3**

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

```

{
  "env": {
    "es6": true,
    "mocha": true,
    "node": true
  },
  "extends": "eslint:recommended",
  "rules": {
    "eol-last": "error",
    "indent": [ "error", 2],
    "no-console": "off",
    "no-multi-spaces": "error",
    "no-trailing-spaces": "error",
    "no-var": "error",
    "prefer-const": "error",
    "quotes": ["error", "single", "avoid-escape"],
    "semi": ["error", "always"],
    "key-spacing": "error",
    "brace-style": "error"
  }
}



```

**What did you do? Please include the actual source code causing the issue.**

``` js

...
      } else {
        text = `${tagName}.${attribs.class}`;
      }
...


```

**What did you expect to happen?**
pass lint rule

**What actually happened? Please include the actual, raw output from ESLint.**
error  Closing curly brace does not appear on the same line as the subsequent block

**Comment by eslintbot** [comment]
Hi @omarcinp, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by platinumazure** [comment]
Hi @omarcinp, thanks for the issue.

Could you please add the rest of the source and return the *full* error message from ESLint (including line and column number)? As it is, I can't tell which brace is being pointed to by the error.

**Comment by vitorbal** [comment]
friendly ping @omarcinp. Could you get back to us with what @platinumazure asks above? Thanks!

**Comment by omarcinp** [comment]
sorry, I cannot share the file, but I'll create another with the same corner case

**Comment by vitorbal** [comment]
@omarcinp sorry to bother you again, was wondering if you had the chance to create a code snippet to reproduce this? Otherwise, I'm afraid the team doesn't have much to work on and we can't help further. Thanks :)

**Comment by vitorbal** [comment]
I'm gonna go ahead and close this since there's not much we can do without more feedback from the OP. We can re-open it if we get more feedback later.
